.github/scripts/requirements.txt

```diff
@@ -0,0 +1 @@
+toml==0.10.2
```

Comment:
There is a `pip install toml` command in `.github/workflows/code_checks.yml`. Should it be pointed to this file or will it be picked up automatically somehow?

Reply:
To get the best of both worlds, I've changed it so that Github Actions will read this new `requirements.txt` file